content/en/docs/concepts/scheduling-eviction/pod-overhead.md

```diff
@@ -122,21 +115,25 @@ For CPU, if the Pod is Guaranteed or Burstable QoS, the kubelet will set `cpu.sh
 requests plus the `overhead` defined in the PodSpec.
 
 Looking at our example, verify the container requests for the workload:
+
 ```bash
 kubectl get pod test-pod -o jsonpath='{.spec.containers[*].resources.limits}'
 ```
 
 The total container requests are 2000m CPU and 200MiB of memory:
+
 ```
 map[cpu: 500m memory:100Mi] map[cpu:1500m memory:100Mi]
 ```
 
 Check this against what is observed by the node:
+
 ```bash
 kubectl describe node | grep test-pod -B2
 ```
 
-The output shows 2250m CPU and 320MiB of memory are requested, which includes PodOverhead:
+The output shows 2250m CPU and 320MiB of memory are requested, which includes Pod overhead:
```

Comment:
How about:
```suggestion
The output shows requests for 2.250 CPU, and for 320MiB of memory. These requests includes Pod overhead:
```

I think “2.25” is more meaningful to people who aren't familiar with Kubernetes quantities or with Docker CPU fractions.

Reply:
I don't think so. The `2250m` expression is the new norm in Kubernetes. People have to get used to that.